src/cursor_ext.rs

```diff
@@ -45,13 +52,71 @@ impl<R: Read + Seek> ReadExt for R {
         }
     }
 
+    fn read_fstring(&mut self) -> Result<Option<String>, Error> {
```

Comment:
is there a reason to implement this out of unreal_helpers and in the library itself?

Reply:
The motivation for changing it in this PR is that the function provided by unreal_helpers is broken. Issue #99 is a regression that was introduced by 2bc021cf32550bb222aef0cfc4f04912ce71eb46, and the easiest fix is to essentially revert the change. It would be feasible to submit a fix to the unreal_helpers project, wait for a release, and then update the version dependency on that crate.  However, the way I see it is that the gvas crate is responsible for de/serializing, so I would prefer to have the code locally. `FString` and `bool` were outliers in that they were the only types that was handled externally. As the implementation for both is trivial, mostly levering behavior from `std`, I think removing the dependency is a good optimization.

I would flip the question back to you and ask: is there a reason that we should take unreal_helpers as a dependency in this crate? It's not clear to me what benefits we get from sharing this code.

Reply:
The reason was to unify common unreal parsing code in a single crate, if that has a bug it would be nice to make a PR fixing it, but I don't see an issue with just having the functions locally here.